scipy/stats/stats.py

```diff
@@ -4425,7 +4435,7 @@ def _euclidean_dist(x):
 
 
 def multiscale_graphcorr(x, y, compute_distance=_euclidean_dist, reps=1000,
-                         workers=1, is_twosamp=False):
+                         workers=1, random_state=None, is_twosamp=False):
```

Comment:
Shouldn't the new keyword argument be added at the end of the parameter list to avoid backward compatibility issues?